support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -427,6 +427,10 @@ footer[role="contentinfo"] a {
   @include mq($from: tablet) {
     margin: ($gu-v-spacing / 4) 0 ($gu-v-spacing / 4 * 3);
   }
+
+  p:not(:last-child) {
```

Comment:
this I also made general. It seems to me if you ever have paragraphs your blurb body, you'll want some spacing between them